### DISCUSSION TITLE
Hexadecimal color support

### DISCUSSION THREAD

**Discussion by Eliott79** [OmerBenGera/WildStacker#352] [Implemented Feature Requests] [answered]
Add hexadecimal color support for item on the floor
![2021-10-13_14 01 43](https://user-images.githubusercontent.com/90210037/137195736-4ce94b24-feba-4d88-b1d3-ea28cf70ebaa.png)

**Comment by Skizzles** [comment, marked as answer]
Was implemented by Omer already.
